Description: Add missing #include <unistd.h> to prevent build failure with GCC4.7.
Bug-Debian: http://bugs.debian.org/cgi-bin/bugreport.cgi?bug=667309
Author: Alessio Treglia <alessio@debian.org>
Forwarded:
---
 demos/FFConsoleDemo.cpp           |    1 +
 demos/OISConsole.cpp              |    1 +
 src/linux/LinuxForceFeedback.cpp  |    1 +
 src/linux/LinuxJoyStickEvents.cpp |    1 +
 4 files changed, 4 insertions(+)

--- ois.orig/src/linux/LinuxJoyStickEvents.cpp
+++ ois/src/linux/LinuxJoyStickEvents.cpp
@@ -30,6 +30,7 @@ restrictions:
 #include "OISEvents.h"
 #include "OISException.h"
 
+#include <unistd.h>
 #include <fcntl.h>        //Needed to Open a file descriptor
 #include <cassert>	
 #include <linux/input.h>
--- ois.orig/src/linux/LinuxForceFeedback.cpp
+++ ois/src/linux/LinuxForceFeedback.cpp
@@ -26,6 +26,7 @@ restrictions:
 #include <cstdlib>
 #include <errno.h>
 #include <memory.h>
+#include <unistd.h>
 
 using namespace OIS;
 
--- ois.orig/demos/OISConsole.cpp
+++ ois/demos/OISConsole.cpp
@@ -12,6 +12,7 @@
 #include <iostream>
 #include <vector>
 #include <sstream>
+#include <unistd.h>
 
 ////////////////////////////////////Needed Windows Headers////////////
 #if defined OIS_WIN32_PLATFORM
--- ois.orig/demos/FFConsoleDemo.cpp
+++ ois/demos/FFConsoleDemo.cpp
@@ -1,5 +1,6 @@
 #include "OIS.h"
 
+#include <unistd.h>
 #include <math.h>
 #include <cstdlib>
 #include <iostream>
